app/src/main/java/com/restaurantroulette/dto/City.kt

```diff
@@ -2,8 +2,8 @@ package com.restaurantroulette.dto
 
 import com.google.gson.annotations.SerializedName
 
-data class City(@SerializedName("cities") var name : String) {
+data class City(@SerializedName("cities") var CityName : String) {
```

Comment:
I'd keep this variable as it was before the code review, for two reasons:
1) Since the class name is City, repeating that in the attribute name is redundant.  The class name gives the "name" variable the correct context to describe it.
2) Per Kotlin Coding Conventions (https://kotlinlang.org/docs/reference/coding-conventions.html), variable/attribute names should be lowerCamelCase, not UpperCamelCase.